src/DiscordBot.py

```diff
@@ -206,13 +207,42 @@ async def on_ready():
     """
     logger.info(f"Logged in as {bot.user.name}")
 
-    github_internship = GitHubUtilities(
-        token=GITHUB_TOKEN, repo_name="SimplifyJobs/Summer2025-Internships", isSummer=True, isCoop=True
+    try:
+        latest_repo = get_latest_internship_repo()
```

Comment:
This should not be called here. This is only triggered when the bot is setting up. It should be in `scheduled_task` when creating a github connection